autogen/agentchat/conversable_agent.py

```diff
@@ -861,13 +860,16 @@ def initiate_chat(
         silent: Optional[bool] = False,
         cache: Optional[Cache] = None,
         max_turns: Optional[int] = None,
+        summary_method: Optional[Union[str, Callable]] = DEFAULT_summary_method,
```

Comment:
Is mixed case var naming in python common?

Reply:
Changed it!